deploy/plays/mysqldb.yml

```diff
@@ -15,10 +15,10 @@
         env:
           MYSQL_ROOT_PASSWORD: "{{ mysql_root_pass }}"
 
-    - name: Give permissions over log directory to mysql user of mysqldb container
```

Comment:
What is the impact of not running this script? MySQL container probably does not start. Please revert this change in mysqldb.yml.